tests/functional/pipelines/designer_test.py

```diff
@@ -117,13 +118,11 @@ def test_routing_pipeline(pipeline):
 
     pipeline.save()
 
-    output = pipeline.run("This is a sample text!")
-
-    assert output["status"] == "SUCCESS"
-    assert output.get("data") is not None
-    assert len(output["data"]) > 0
-    assert output["data"][0].get("segments") is not None
-    assert len(output["data"][0]["segments"]) > 0
+    output = pipeline.run("This is a sample text!", **{"batchmode": False, "version": "3.0"})
+    print("output")
+    print(output)
+    print(output.status)
```

Comment:
many prints here as well, better to check the rest of the work in case any redundant print statements.